src/js/core/grid.js

```
@@ -1,136 +1,210 @@
                 }
 
-                rows.forEach((row, i) =>
-                    row.forEach((el, j) =>
-                        css(el, 'transform', !scrolled && !translates ? '' : `translateY(${
-                            (translates && -translates[i][j]) + (scrolled ? j % 2 ? scrolled : scrolled / 8 : 0)
-                        }px)`)
-                    )
-                );
+                columns.forEach((column, i) =>
+                    column.forEach((el, j) => {
+                        let [x, y] = (translates && translates[i][j]) || [0, 0];
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)